Px.Utils.UnitTests/ModelBuilderTests/MatrixMetadataBuilderTests.cs

```diff
@@ -525,5 +528,62 @@ public void MultilanguageTableWithCustomMetaPropertiesAndPropetyTypes()
             Assert.AreEqual(MetaPropertyType.TextArray, actual.AdditionalProperties["SINGLEITEMTEXTARRAYPROPERTY"].Type);
             Assert.AreEqual(MetaPropertyType.MultilanguageTextArray, actual.AdditionalProperties["SINGLEITEMMULTILANGUAGETEXTARRAYPROPERTY"].Type);
         }
+
+        [TestMethod]
+        public void MultilanguageRemovesDimensionTypeAndTimeValEntriesTest()
+        {
+            Assert.IsTrue(Actual_1Lang_With_Range_Time_Dimension.Dimensions.Exists(dim => dim.Type == DimensionType.Time));
+            foreach (Dimension dim in Actual_3Lang.Dimensions)
+            {
+                Assert.IsFalse(dim.AdditionalProperties.ContainsKey(PxFileConfiguration.Default.Tokens.KeyWords.DimensionType));
+
+                if (dim.Type == DimensionType.Time)
+                {
+                    Assert.IsTrue(dim.AdditionalProperties.TryGetValue(PxFileConfiguration.Default.Tokens.KeyWords.TimeVal, out MetaProperty? value));
+                    Assert.AreEqual(MetaPropertyType.TextArray, value.Type);
+                    StringListProperty property = (StringListProperty)value;
+                    Assert.AreEqual(8, property.Value.Count);
+                }
+            }
+        }
+
+        [TestMethod]
+        public void MultilanguageWithRangeTimeDimensionTest()
+        {
+            Assert.IsTrue(Actual_1Lang_With_Range_Time_Dimension.Dimensions.Exists(dim => dim.Type == DimensionType.Time));
+            foreach (Dimension dim in Actual_1Lang_With_Range_Time_Dimension.Dimensions)
+            {
+                Assert.IsFalse(dim.AdditionalProperties.ContainsKey(PxFileConfiguration.Default.Tokens.KeyWords.DimensionType));
+
+                if (dim.Type == DimensionType.Time)
+                {
+                    Assert.IsTrue(dim.AdditionalProperties.ContainsKey(PxFileConfiguration.Default.Tokens.KeyWords.TimeVal));
+
+                    if (dim.Type == DimensionType.Time)
+                    {
+                        Assert.IsTrue(dim.AdditionalProperties.TryGetValue(PxFileConfiguration.Default.Tokens.KeyWords.TimeVal, out MetaProperty? value));
+                        Assert.AreEqual(MetaPropertyType.Text, value.Type);
+                        StringProperty property = (StringProperty)value;
+                        Assert.AreEqual("2015-2022", property.Value);
+                    }
+                }
+            }
+        }
+
+        [TestMethod]
+        public void MultilanguageRemovesTableLevelMetaEntriesTest()
+        {
+            string[] keywords = [
+                PxFileConfiguration.Default.Tokens.KeyWords.Units,
+                PxFileConfiguration.Default.Tokens.KeyWords.ShowDecimals,
```

Comment:
Maybe leave show decimals? It is/should be presentation level information and not directly overlapping with the content variable.

Reply:
SHOWDECIMALS now retains after building content dimension